drivers/espi/Kconfig

```diff
@@ -0,0 +1,98 @@
+# Kconfig - DMA configuration options
+#
+#
+# Copyright (c) 2016 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# ESPI options
+#
+menuconfig ESPI
+	bool "ESPI Driver"
+	help
+	  Enable ESPI Driver
+
+if ESPI
+
+module = ESPI
+module-str = espi
+source "subsys/logging/Kconfig.template.log_config"
+
+config ESPI_SLAVE
+	bool "ESPI slave driver"
+	default y
+	help
+		Enables eSPI driver in slave mode
+
+config ESPI_INIT_PRIORITY
+	int "IRQ Priority for ESPI Controller"
+	default 3
+	help
+		IRQ Priority for ESPI Controller.
+
+config ESPI_CHANNEL_PERIPHERAL
+	bool "ESPI peripheral channel"
+	default y
+	help
+		ESPI Controller supports peripheral channel.
+
+config ESPI_CHANNEL_VWIRE
+	bool "ESPI flash channel"
+	default y
+	help
+		ESPI Controller supports virtual wires channel.
+
+config ESPI_CHANNEL_OOB
+	bool "IRQ Priority for ESPI Controller"
+	default n
+	help
+		ESPI Controller supports OOB channel.
+
+if ESPI_CHANNEL_PERIPHERAL
+
+config ESPI_CHANNEL_FLASH
+	bool "ESPI flash channel"
+	default n
+	help
+		ESPI Controller supports flash channel.
+
+config ESPI_PERIPHERAL_UART
+	bool "UART"
```

Comment:
"UART peripheral"

Add "peripheral" to all relevant option titles below as well.